smart_control/reinforcement_learning/scripts/generate_gin_config_files.bash

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Run the generator script
```

Comment:
My inclination would be to add these commands to one or more README file(s) instead of bash files, especially if they reference a specific user's filepaths.